docs/kubernetes/kctlr-manage-bigip-objects.rst

```diff
@@ -1,317 +1,204 @@
 
-#. Upload the ConfigMap to the Kubernetes/OpenShift API server.
-
-   .. code-block:: console
-      :caption: kubectl
-
-      kubectl create -f f5-resource-pool-only-example.configmap.yaml [--namespace=<service-namespace>]
-      configmap "http.pool_only" created
+#. :ref:`Update the API server <kctlr upload resource api server>`.
 
-   .. code-block:: console
-      :caption: openshift cli
+#. Use the BIG-IP configuration utility or TMSH to add the pool members to an iRule or traffic policy with the correct routing rules for your application **--OR--**
 
-      oc create -f f5-resource-pool-only-example.configmap.yaml [--namespace=<service-namespace>]
-      configmap "http.pool_only" created
-
-.. important::
+   :ref:`kctlr-attach-pool-vs` **--OR--**
 
-   Don't forget to attach the pool to the iRule or traffic policy on the BIG-IP system that knows how to route traffic for the Service.
+   :ref:`kctlr-ipam`.
 
-.. _kctlr-attach-pool-vs:
+.. _kctlr-ipam:
 
-Attach a pool to an existing virtual server
-```````````````````````````````````````````
+Attach pools to virtual servers using IPAM
+``````````````````````````````````````````
 
-#. Add the desired :code:`bindAddr` to the F5 virtual server resource definition.
+.. include:: /_static/reuse/k8s-version-added-1_1.rst
 
-   .. tip::
+One benefit of unattached pools is the compatibility with IP address allocation using IPAM.
+To add the IP address allocated by an IPAM system to an F5 Resource ConfigMap, use an `Annotation`_ with the following format:
 
-      You can configure your IPAM system to do this automatically. For example:
+:code:`virtual-server.f5.com/ip=<ip_address>`
 
-      :code:`kubectl annotate configmap http.pool_only virtual-server.f5.com/ip=1.2.3.4`
+When you annotate the F5 resource, the |kctlr| does the following:
 
-#. Verify the changes using :command:`kubectl get` or :command:`oc get`.
+- creates a new BIG-IP virtual server with the designated IP address, and
+- attaches the pool to the virtual server.
 
-   .. code-block:: console
+To provide even greater benefit, consider automating this process from your IPAM system.
+Here's a high-level example:
 
-      kubectl get configmap http.pool_only -o yaml \\
-      oc get configmap http.pool_only -o yaml
+#. Watch the Kubernetes/OpenShift or BIG-IP API for creation of ConfigMaps that
 
-#. Go to :menuselection:`Local Traffic --> Virtual Servers` in the BIG-IP configuration utility to verify the pool attached to the virtual server. (Be sure to look in the correct BIG-IP partition.)
+   - have :code:`f5type: virtual-server` **AND**
+   - don't have a :code:`frontend.virtualAddress.bind-addr` entry.
 
+#. Allocate an IP address for the unattached pool.
+#. Use the allocated IP address to annotate the F5 resource ConfigMap.
 
-.. _kctlr-delete-unattached-pool:
+.. _kctlr-attach-pool-vs:
 
-Delete an "unattached" pool
-```````````````````````````
+Attach a pool to an existing virtual server
```

Comment:
I think this section should be deleted. I'm not sure how this works.
1) If trying to use an existing virtual with a pool already attached we'll get errors.
2) If trying to use an existing virtual in /Common we'll get conflict errors as the CC will try and create a virtual in the managed partition.
3) If you create a virtual in the managed partition it will be deleted.